lib/sshutils/server_test.go

```diff
@@ -25,74 +25,82 @@ import (
 
 	"golang.org/x/crypto/ssh"
 
-	"github.com/gravitational/teleport/lib/utils"
-
 	"github.com/gravitational/trace"
 
-	"gopkg.in/check.v1"
+	"github.com/gravitational/teleport/lib/utils"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
 )
 
 func TestMain(m *testing.M) {
 	utils.InitLoggerForTests()
 	os.Exit(m.Run())
 }
 
-func TestSSHUtils(t *testing.T) { check.TestingT(t) }
-
-type ServerSuite struct {
-	signer ssh.Signer
-}
-
-var _ = check.Suite(&ServerSuite{})
+func TestStartStop(t *testing.T) {
+	t.Parallel()
 
-func (s *ServerSuite) SetUpSuite(c *check.C) {
-	var err error
-	_, s.signer, err = utils.CreateCertificate("foo", ssh.HostCert)
-	c.Assert(err, check.IsNil)
-}
+	_, signer, err := utils.CreateCertificate("foo", ssh.HostCert)
+	require.NoError(t, err)
 
-func (s *ServerSuite) TestStartStop(c *check.C) {
 	called := false
 	fn := NewChanHandlerFunc(func(_ context.Context, _ *ConnectionContext, nch ssh.NewChannel) {
 		called = true
-		err := nch.Reject(ssh.Prohibited, "nothing to see here")
-		c.Assert(err, check.IsNil)
+
+		if err := nch.Reject(ssh.Prohibited, "nothing to see here"); err != nil {
+			logrus.Errorf("Failed to reject connection: %v", err)
+		}
```

Comment:
```suggestion
		err := nch.Reject(ssh.Prohibited, "nothing to see here")
		assert.NoError(t, err)
```

from `github.com/stretchr/testify/assert`